src/URLFixer.php

```diff
@@ -1,70 +1,66 @@
 <?php
 
-namespace Roots\Bedrock;
+declare(strict_types=1);
+
+namespace Yard\Bedrock;
 
 /**
  * Class URLFixer
- * @package Roots\Bedrock
+ *
+ * @package Yard\Bedrock
+ *
  * @author Roots
+ *
  * @link https://roots.io/
  */
 class URLFixer
 {
-    /**
-     * Add filters to verify / fix URLs.
-     */
-    public function addFilters()
-    {
-        add_filter('option_home', [$this, 'fixHomeURL']);
-        add_filter('option_siteurl', [$this, 'fixSiteURL']);
-        add_filter('network_site_url', [$this, 'fixNetworkSiteURL'], 10, 3);
-    }
+	/**
+	 * Add filters to verify / fix URLs.
+	 */
+	public function addFilters(): void
+	{
+		add_filter('option_home', [$this, 'fixHomeURL']);
+		add_filter('option_siteurl', [$this, 'fixSiteURL']);
+		add_filter('network_site_url', [$this, 'fixNetworkSiteURL'], 10, 3);
```

Comment:
Je zou dit nog kunnen aanpassen naar de first class callable syntax, schijnt ook sneller te zijn.

Reply:
Alleen verhoog je daarmee de minimale php versie naar 8.1. Voor nu laat ik het zo.

Reply:
Waar draaien we dan 8.0?

Reply:
https://github.com/yardinternet/multisite-url-fixer/blob/master/composer.json#L19

Reply:
Je verwijst naar php 7.4, draaien er bedrock sites van ons op 7.4 dan?